changelog/install-dmd-dc.dd

```diff
@@ -0,0 +1,27 @@
+`~/dlang/install.sh` now supports `--dmd` and `--dc`
```

Comment:
These are not options, they're actions. They should be named "get-dmd-path" and "get-dc-path" OSLT.

Am I missing something?